tests/test_installer.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2024 Science project contributors.
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import subprocess
+
+from _pytest.tmpdir import TempPathFactory
+
+INSTALLER = "./install.sh"
```

Comment:
This will work fine, but FYI there is a build_root Path fixture in conftest.

Reply:
thanks, moved to that